std/tasks/simplelog/logger.go

```diff
@@ -38,6 +53,24 @@ func (sl *logger) Waiting(ra *tasks.RunningAction) {
 	// Do nothing.
 }
 
+func (sl logger) write(b []byte) {
+	if LogActionsToFile == "" {
+		// Ignore errors
+		_, _ = sl.out.Write(b)
+		return
+	}
+
+	fo, err := os.OpenFile(LogActionsToFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
What about opening the file on NewSink, and set sl.out?

Reply:
I tried this first. My impression is that there is an ordering problem (we call NewSink before evaluating flags).